src/quick-lint-js/parse.h

```diff
@@ -1549,6 +1555,9 @@ class parser {
         method_attributes = function_attributes::async_generator;
         this->skip();
       }
+      if (this->peek().type == token_type::kw_static) {
+        async_static = true;
```

Comment:
Document with a comment:
```suggestion
      if (this->peek().type == token_type::kw_static) {
        // async static method() {}  // Invalid.
        // async static() {}
        async_static = true;
```